packages/addon-dev/src/rollup-public-entrypoints.ts

```diff
@@ -16,14 +16,14 @@ export default function publicEntrypoints(args: {
   return {
     name: 'addon-modules',
     async buildStart() {
-      this.addWatchFile(args.srcDir);
-
       let matches = walkSync(args.srcDir, {
         globs: [...args.include, '**/*.hbs', '**/*.ts', '**/*.gts', '**/*.gjs'],
         ignore: args.exclude,
       });
 
       for (let name of matches) {
+        this.addWatchFile(path.join(args.srcDir, name));
```

Comment:
what does this mean for css resources/images and paths that are not matched here?

Reply:
@void-mAlex, if you just want to move some static assets to `dist` folder, then you should relly on the `rollup-keep-assets` file, but if you are talking about for example importing css or images inside of js files then it should be up to which ever rollup plugin you are using for that to add the css/image to the watched list.

For example, let's say you are doing
```
import styles from './foo.css'
```
And using [rollup-plugin-postcss](https://www.npmjs.com/package/rollup-plugin-postcss) to build this, in that case it is `rollup-plugin-postcss`'s job to include the css file to the watch list, which is does [here](https://github.com/egoist/rollup-plugin-postcss/blob/master/src/index.js#L124) I believe